queries/futures/useGetFuturesCumulativeStats.ts

```diff
@@ -9,12 +9,16 @@ import QUERY_KEYS from 'constants/queryKeys';
 import { FuturesCumulativeStats } from './types';
 import { getFuturesEndpoint } from './utils';
 import { wei } from '@synthetixio/wei';
+import { FUTURES_ENDPOINT_MAINNET } from './constants';
 
-const useGetFuturesCumulativeStats = (options?: UseQueryOptions<FuturesCumulativeStats | null>) => {
+const useGetFuturesCumulativeStats = (
+	homepage?: boolean,
```

Comment:
Just a nit, but maybe we can make a hook that matches the URL to check if we're on the homepage, so we can reuse it. Nothing wrong with this approach though, just think it might be useful for other things in the future as well. I see that you used a similar approach in the `useGetFuturesDailyTradeStats` hook and other hooks.

Reply:
On second thought, instead of manually checking if the user is on the homepage, we could just use the Infura endpoint when a user is on an unsupported network. We should be able to do that from within the `getFuturesEndpoint`, I think.